release_notes/Service_Fabric_ReleaseNotes_10.md

```diff
@@ -0,0 +1,89 @@
+# Microsoft Azure Service Fabric 10.0 Release Notes
+
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Key Announcements](#key-announcements)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Breaking Changes](#breaking-changes)
```

Comment:
Breaking Changes should be ahead of Features & Bug Fixes. Place it between Key Announcements & Features and Bug Fixes.